src/components/AttachmentCarousel/index.js

```diff
@@ -265,10 +265,7 @@ class AttachmentCarousel extends React.Component {
 
     /**
      * Defines how a single attachment should be rendered
-     * @param {Object} item
-     * @param {String} item.source
-     * @param {Object} item.file
-     * @param {String} item.file.name
+     * @param {{ isAuthTokenRequired: Boolean, source: String, file: { name: String } }} item
```

Comment:
NAB: This is against the convention in our style guide https://github.com/Expensify/App/blob/ac3ac1f1257d3224fe206030e66cb89bacf46910/contributingGuides/STYLE.md#L165